src/components/tabs/components/tab-button-controls/tab-button-controls.module.css

```diff
@@ -33,7 +33,7 @@ the correct elements, roles, states, and properties are being set in the markup.
 Styles for each tab button.
 */
 .tabButton[role='tab'] {
-	background-color: var(--token-color-surface-primary);
```

Comment:
I'm not sure of the history of why this was set to 'white' and the other variants set to 'transparent'. Setting all to 'transparent' seems to have the same impact for current light mode. Please let me know if I'm missing something here!